Server/api/routes/user.ts

```diff
@@ -1,10 +1,76 @@
 import Router from "@koa/router";
-import { isLoggedIn } from "../../middleware";
+import { User } from "../../../Models/user";
+import { UsernameChange } from "../../../Models/usernameChange";
+import { isHeadStaff, isLoggedIn } from "../../middleware";
 
 const userRouter = new Router();
 
 userRouter.get("/", isLoggedIn, async (ctx) => {
     ctx.body = await ctx.state.user.getInfo();
 });
 
+userRouter.post("/username/add", isHeadStaff, async (ctx) => {
+    const body = ctx.request.body;
+    if (!body.ID || !body.username)
+        return ctx.body = { 
+            error: "Missing parameters",
+        };
+    
+    const user = await User.findOneOrFail({
+        ID: body.ID,
+    });
+
+    if (user.osu.username)
+        return ctx.body = { 
+            error: "User already has a username",
+        };
+
+    user.osu.username = body.username;
+    await user.save();
+    ctx.body = {
+        success: true,
+        user,
+    };
+});
```

Comment:
if i understand right, this is due to an issue with the seeding script which i'd rather fix directly instead, and fix existing bogus entries via SQL. can you specify issues you've seen so i can look at fixing them?

Reply:
Oh yea toara and Segula are missing usernames in the DB currently cuz they are banned, osu user IDs 12911800 and 10414232 respectively

Reply:
i should have fixed the root cause with https://github.com/Corsace/Corsace/pull/147 and can fix these entries via SQL.

i don't think we should keep this route, considering maintenance costs and also the fact that it's incomplete (missing adding username to history).

Reply:
Ok should i also remove the delete route as well then, might just be better to also remove those instances via SQL as well